Orbitersdk/samples/ProjectApollo/src_csm/saturn.h

```diff
@@ -1289,6 +1289,7 @@ class Saturn: public ProjectApolloConnectorVessel, public PanelSwitchListener {
 	// \ mass in kg, ro (distance from center of the bottom points), tdph (height of bottom points),
 	// \ height (of the top point), x_target (stiffness/damping factor, stable default is -0.5)
 	//
+	void ConfigTouchdownPoints();
 	void ConfigTouchdownPoints(double mass, double ro, double tdph, double height, double x_target = -0.5);
```

Comment:
ConfigTouchdownPoints gets called from the normal SetCSMStage function but now also from UpdateMassAndCoG, as touchdown points aren't automatically shifted with ShiftCG. So I just wanted to implement a common function that can be called from both places and I didn't want to have a duplicate of the fixed input parameters, so I made a version of ConfigTouchdownPoints without input parameters.

And I made it a switch/case because the function might be used by other stages in the future. Only by the CSM stage for now though.

Reply:
Okay! Whenever its logical the rest of the stages can be moved to the new function. Fine for now.